### DISCUSSION TITLE
Can i add an default side?

### DISCUSSION THREAD

**Discussion by loopsFreitag** [shortcuts/no-neck-pain.nvim#305] [Q&A] [answered]
I want always have this plugin running so i enabled on autocmd
```
{
    "shortcuts/no-neck-pain.nvim",
    config = function ()
      require("no-neck-pain").setup({
        autocmds = {
          enableOnVimEnter = true,
          enableOnTabEnter = true,
        }
      })
    end,
  },

```

but i want to use the buffer on right side

i tried to add an auto command to always apply it, but it runs before the plugin configuration
```
{
    "VimEnter", {
      pattern = { "*" },
      command = "NoNeckPainToggleRightSide"
    }
  }
```
so i want to know, theres an way to do that natively? or i have to add auto commands for that?

**Comment by shortcuts** [comment, marked as answer]
Hey, thanks for using the plugin :)

> but i want to use the buffer on right side

Do you mean you'd like to only have one of the side enabled? 

The following configuration will only enable **the left side**

```
{
    autocmds = {
        enableOnVimEnter = true,
        enableOnTabEnter = true,
    },
    buffers = {
        right = {
            enabled = false,
        },
    },
}
```

The following configuration will only enable **the right side**

```
{
    autocmds = {
        enableOnVimEnter = true,
        enableOnTabEnter = true,
    },
    buffers = {
        left = {
            enabled = false,
        },
    },
}
```

**Reply by loopsFreitag** [reply]
That was it, thanks

**Reply by shortcuts** [reply]
Enjoy!!
